sdk/avalon_sdk/tcf_connector.toml

```diff
@@ -79,3 +79,9 @@ gas_price = "100"
 [fabric]
 fabric_network_file = "sdk/avalon_sdk/fabric/network.json"
 channel_name = "mychannel"
+
+[WorkerConfig]
+# These configs need to be same as that in tcs_config -
```

Comment:
If they have to be the same, can't the program read them directly from c`onfig/tcs_config.toml` ?

Reply:
The connector config has different set of configs specific to connector. So, it reads from a separate config file. It would not be a good approach to pass in 2 config files. Though these configurations might themselves be moved shortly because the connector will not handle worker sync-up. There would be a separate component doing this which would need the config. For now this is an added responsibility of the connector and hence the configs.